types/express-async-errors/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for express-async-errors 3.1
+// Project: https://github.com/davidbanham/express-async-errors
+// Definitions by: Stephen Melvin <https://github.com/sbmelvin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const _: object;
+export = _;
```

Comment:
```suggestion
export {};
```

Reply:
Auto-imports would pick this up as a suggestion for when the user types `_` 😉

Reply:
Thanks for the feedback.

Reply:
So that change seems to cause the tests to fail:
```
Error: /Users/stephen/Development/DefinitelyTyped/types/express-async-errors/express-async-errors-tests.ts:2:1
ERROR: 2:1  expect  TypeScript@4.7 expected type to be:
  object
got:
  typeof import("/Users/stephen/Development/DefinitelyTyped/types/express-async-errors/index")
```

Reply:
Yeah but you wrote that test 😆

Reply:
I don’t think there are any meaningful tests to write for these “typings”

Reply:
Actually, TS doesn’t complain if you `import "express-async-errors"` even if it can’t find any typings for it—so why are we adding it?

Reply:
It depends on the config. `noImplicitAny: true` results in the error:
```
Could not find a declaration file for module 'express-async-errors'. './node_modules/express-async-errors/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/express-async-errors` if it exists or add a new declaration (.d.ts) file containing `declare module 'express-async-errors';`ts(7016)
```

Reply:
You would get that if you do `import {} from "express-async-errors"` or `import whatever = require("express-async-errors")` but not if you do `import "express-async-errors"`

Reply:
`import "express-async-errors"` is a much better solution, thanks! I no longer think this definition is needed.